pom.xml

```diff
@@ -792,6 +793,16 @@
                         </pomElements>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.basepom.maven</groupId>
+                    <artifactId>inline-maven-plugin</artifactId>
```

Comment:
Cool! Looks like the repository is not available? https://github.com/basepom/inline-maven-plugin

Reply:
oops. It is public now. I kept it private for a while because the code changed a lot.